django/thunderstore/repository/models/comment.py

```diff
@@ -0,0 +1,40 @@
+import uuid
+
+from django.conf import settings
+from django.contrib.contenttypes.fields import GenericForeignKey
+from django.contrib.contenttypes.models import ContentType
+from django.db import models
+from ulid2 import generate_ulid_as_uuid
+
+
+class Comment(models.Model):
+    commented_object = GenericForeignKey()
```

Comment:
Having thought about this naming a bit more, `thread` would seem most intuitive naming to me personally so let's go with that unless you strongly feel otherwise 😄